pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/pdf/DownloadPDF.java

```diff
@@ -136,7 +136,7 @@ private Optional<byte[]> generateSinglePDF(@NonNull String requestBody, String c
                 break;
             } else {
                 try {
-                    logger.error(String.format("Got %s PDF with size zero in try %d", configName, turn));
```

Comment:
We created this error last year to make sure this condition is real and is happening often in prod. We have now observed the errors and don't need it anymore